frame/staking/src/pallet/impls.rs

```diff
@@ -59,6 +59,13 @@ use super::{pallet::*, STAKING_ID};
 const NPOS_MAX_ITERATIONS_COEFFICIENT: u32 = 2;
 
 impl<T: Config> Pallet<T> {
+	/// Returns the current nominations quota for nominators.
+	///
+	/// Used by runtime API.
+	pub fn api_nominations_quota() -> u32 {
+		T::MaxNominations::get()
```

Comment:
Is the plan that right now it will just return a constant value but later we will add more logic (once your PR is merged) such that if the runtime uses a Dynamic Nominations Quota, it will return a dynamic value? Since it not very useful right now, should we add this later (when we need it)?

Reply:
The plan is to start preparing the community to use this runtime API before the dynamic nominations is released, so that the transitions is smooth. I will add the `balance` as input to the runtime API and RPC so that the API doesn't change anymore.